src/components/common/Footer/Footer.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+
+const Footer = () => {
+    return (
+        <div>
+        <footer className="footer" style={{
+            marginTop: "1rem",
+            padding: "1rem",
+            backgroundColor: "white",
+            borderTop: "1px solid #808080",
+            position: "absolute",
+            textAlign: "center",
+            bottom: 0,
+            left: 0,
+            width: "100%"
+        }}>
+    <p style={{color: "#808080"}}>© 2021 Crunchyroll GirlsGoIT</p>
```

Comment:
Please use Text component

Reply:
Isn't that react native? You mean  <Text>?

Reply:
We use React library as we are working on the web application. The react-native library is used for mobile apps (iOS and Android)